### DISCUSSION TITLE
NEW DISCORD BOT

### DISCUSSION THREAD

**Discussion by coderboi64** [Rapptz/discord.py#6349] [Ideas] [answered]
Hello, my name is Kaleb or on discord flight is ok#6348 and I have an idea for a new bot but I need some help I don't really care if the person helping is a professional or not I just need some help please and thank you.

**Comment by mesub7** [comment, marked as answer]
Hey there.

This is not the place to ask for complete snippets or asking people to write your bot (or portions of it) for you. You are expected to put in some effort and provide evidence of this (for example your code or a specific problem that would help them complete your implementation).


Take care.
